test/pyro/test_convert.py

```diff
@@ -105,7 +105,7 @@ def _rand(batch_shape, *event_shape):
     # Evaluate via two different patterns.
     expected = f(x=value_x)(y=value_y)
     actual = f(y=value_y)(x=value_x)
-    assert_close(actual, expected, atol=1e-5, rtol=1e-5)
+    assert_close(actual, expected, atol=1e-5, rtol=2e-4)
```

Comment:
a 20x error increase seems like a lot, any idea why?

Reply:
I also found this puzzling since it wasn't clear how the changes in this PR would affect `AffineNormal`, maybe it's due to nondeterminism somewhere in the test setup?